2-rolling-upgrade/README.md

```diff
@@ -69,7 +72,9 @@ First get the current version of each broker in the cluster and verify that brok
 
 ```
 rpk redpanda admin brokers list
-```
+```{{exec}}
```

Comment:
Since this is the README it needs to exclude `{{exec}}`, as that's only for killercoda and it messes up the formatting when viewing this README on Github.